crates/sui-json-rpc/src/api/indexer.rs

```diff
@@ -92,4 +92,14 @@ pub trait IndexerApi {
         /// The Name of the dynamic field
         name: DynamicFieldName,
     ) -> RpcResult<SuiObjectResponse>;
+
+    /// Return the resolved address given resolver and name
+    #[method(name = "resolveNameServiceAddress")]
+    async fn resolve_name_service_address(
+        &self,
+        /// The resolver address
```

Comment:
I thought the idea was to bake this into the fullnode itself and not make it configurable per the request?